### PR TITLE
DOCS-926: Update single axis and multi axis configs

### PR DESCRIPTION
**Pull request by martha-johnston**
This PR updates the `single-axis` and `multi-axis` docs to have accurate configs and attributes

### CODE REVIEW

File: docs/components/gantry/multi-axis.md [hunk 1]
```diff
@@ -209,3 +210,4 @@ The following attributes are available for `multi-axis` gantries:
 | Attribute | Type | Inclusion | Description |
 | --------- | ---- | --------- | ----------- |
 | `subaxes_list` | array | **Required** | An array of the `name` of each of the sub-axes, the [`single-axis`](/components/gantry/single-axis/) gantries that make up the `multi-axis` gantry. |
+| `move_simultaneously` | boolean | Optional | An boolean indicating if the sub-axes should move together, or one at a time when `MoveToPosition` is called. |
```
Comment (npentrel) [inline docs/components/gantry/multi-axis.md:213]:
```suggestion
| `move_simultaneously` | boolean | Optional | A boolean indicating if the sub-axes should move together or one at a time when `MoveToPosition` is called. |
```

File: docs/components/gantry/single-axis.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ description: "Configure a single-axis gantry."
 images: ["/icons/components/gantry.svg"]
 tags: ["gantry", "components"]
 aliases:
-    - "/components/gantry/oneaxis/"
-# SME: Rand
+    - "/components/gantry/singleaxis/"
```
Comment (npentrel) [inline docs/components/gantry/single-axis.md:10]: Please undo this change. This was correct as is. The page was called this before and therefore we need a redirect

File: docs/components/gantry/single-axis.md [hunk 3]
```diff
@@ -61,9 +57,8 @@ The following attributes are available for `single-axis` gantries:
...
 | `board`  |  Optional | string | `name` of the [board](/components/board/) containing the limit switches and pins. If `limit_pins` exist, `board` is required. |
 | `limit_pins`  | Optional | object | The `boards`'s pins attached to the limit switches on either end. If the [motor](/components/motor/) used does not include an [encoder](/components/motor/gpio/encoded-motor/), `limit_pins` are required to be set. |
 | `limit_pin_enabled_high` | boolean | Optional | Whether the limit pins are enabled. <br> Default: `false` |
-| `mm_per_rev` | int | Optional | How far the gantry moves (linear, distance in mm) per one revolution of the motor’s output shaft. This typically corresponds to Distance = PulleyDiameter * pi, or the pitch of a linear screw. |
-| `gantry_rpm` | int | Optional | The gantry `motor`'s default revolutions per minute (RPM). |
+| `mm_per_rev` | int | **Required** | How far the gantry moves (linear, distance in mm) per one revolution of the motor’s output shaft. This typically corresponds to Distance = PulleyDiameter * pi, or the pitch of a linear screw. |
+| `gantry_mm_per_sec` | int | Optional | The gantry `motor`'s default revolutions per minute (RPM). |
```
Comment (randhid) [inline docs/components/gantry/single-axis.md:64]:
```suggestion
| `gantry_mm_per_sec` | int | Optional | The speed at which the gantry moves in millimeters per second. Used to calculate the gantry `motor`'s revolutions per minute (RPM). It defaults to 100. |
```

**Review by randhid — APPROVED** [review]
LGTM pending docs re-review.

File: docs/components/gantry/single-axis.md [hunk 4]
```diff
@@ -61,9 +57,8 @@ The following attributes are available for `single-axis` gantries:
...
 | `limit_pins`  | Optional | object | The `boards`'s pins attached to the limit switches on either end. If the [motor](/components/motor/) used does not include an [encoder](/components/motor/gpio/encoded-motor/), `limit_pins` are required to be set. |
 | `limit_pin_enabled_high` | boolean | Optional | Whether the limit pins are enabled. <br> Default: `false` |
-| `mm_per_rev` | int | Optional | How far the gantry moves (linear, distance in mm) per one revolution of the motor’s output shaft. This typically corresponds to Distance = PulleyDiameter * pi, or the pitch of a linear screw. |
-| `gantry_rpm` | int | Optional | The gantry `motor`'s default revolutions per minute (RPM). |
+| `mm_per_rev` | int | **Required** | How far the gantry moves (linear, distance in mm) per one revolution of the motor’s output shaft. This typically corresponds to Distance = PulleyDiameter * pi, or the pitch of a linear screw. |
+| `gantry_mm_per_sec` | int | Optional | The speed at which the gantry moves in millimeters per second. Used to calculate the gantry `motor`'s revolutions per minute (RPM). It defaults to 100 RPM. |
```
Comment (sguequierre) [inline docs/components/gantry/single-axis.md:64]:
```suggestion
| `gantry_mm_per_sec` | int | Optional | The speed at which the gantry moves in millimeters per second. Used to calculate the gantry `motor`'s revolutions per minute (RPM). <br> Default: `100` |
```
for docs formatting-- unless (q) I'm confused-- is the value itself a derivative of some kind to get 100 rpm?

File: docs/components/gantry/multi-axis.md [hunk 5]
```diff
@@ -209,3 +210,4 @@ The following attributes are available for `multi-axis` gantries:
 | Attribute | Type | Inclusion | Description |
 | --------- | ---- | --------- | ----------- |
 | `subaxes_list` | array | **Required** | An array of the `name` of each of the sub-axes, the [`single-axis`](/components/gantry/single-axis/) gantries that make up the `multi-axis` gantry. |
+| `move_simultaneously` | boolean | Optional | A boolean indicating if the sub-axes should move together, or one at a time when `MoveToPosition` is called. |
```
Comment (randhid) [inline docs/components/gantry/multi-axis.md:213]: To amtch docs' styling:
```suggestion
| `move_simultaneously` | boolean | Optional | A boolean indicating if the sub-axes should move together, or one at a time when `MoveToPosition` is called. <br> Default:  `false` |
```

File: docs/components/gantry/single-axis.md (hunk 4 above)
Comment (martha-johnston) [reply docs/components/gantry/single-axis.md:64]: yeah whatever value the user puts in we convert to RPM because that's what the motor needs, so the default basically whatever equals 100 rpm, but that will be different based on each users setup

Comment (martha-johnston) [reply docs/components/gantry/single-axis.md]: added RPM to your suggestion... does that work @sguequierre?

**Comment by npentrel** [comment]
Updated the commits - we've made some changes to how images work in the docs. Feel free to merge when you're ready

**Review by npentrel — APPROVED** [review]
